sources/packages/web/src/views/aest/NoticeOfAssessment.vue

```diff
@@ -9,17 +9,21 @@
           params: { applicationId, studentId, assessmentId },
         }"
       />
-    </div>
-  </v-container>
-  <full-page-container>
-    <notice-of-assessment-form-view :assessmentId="assessmentId" />
+    </template>
+    <notice-of-assessment-form-view
+      :assessmentId="assessmentId"
+      :can-reissue-m-s-f-a-a="true"
+      @reissue-m-s-f-a-a="reissueMSFAA"
```

Comment:
Not a problem, just curious: should we use kebab case for the attributes too? If that's case `assessmentId` should be `assessment-id` 😄

Reply:
yes 😄 we can do it and make sense